test/__fixtures__/workspace/download-error3.txt

```diff
@@ -0,0 +1,5 @@
+You have not selected any the of following - Workspace
+You are currently in:
+1. Org: <no org selected>
```

Comment:
doesn't make sense. error doesn't show Org, Project, but Org and Workspace config shows is 'no x selected'